graphql/src/Mu/GraphQL/Query/Parse.hs

```diff
@@ -659,52 +660,59 @@ class ValueParser (sch :: Schema') (v :: FieldType Symbol) where
 
 instance ValueParser sch 'TNull where
   valueParser _ _ GQL.Null = pure FNull
-  valueParser _ fname _
-    = throwError $ "field '" <> fname <> "' was not of right type"
+  valueParser _ fname _    = throwError $ "field '" <> fname <> "' was not of right type"
```

Comment:
Not great that the style changed, this makes the diff a bit bigger than it should :/